README.developer.md

```diff
@@ -181,55 +181,55 @@ meson install -C builddir --destdir bin
 Meson will output the artifacts to `./builddir/bin/usr/local/`. This directory is referred to in the following sections
 simply as `<builddir>`.
 
-To allow `hirte` and `hirte-agent` to own a name on the local system D-Bus, the provided configuration
+To allow `bluechi` and `bluechi-agent` to own a name on the local system D-Bus, the provided configuration
 files need to be copied (if not already existing):
 
 ```bash
-cp <builddir>/share/dbus-1/system.d/org.containers.hirte.Agent.conf /etc/dbus-1/system.d/
-cp <builddir>/share/dbus-1/system.d/org.containers.hirte.conf /etc/dbus-1/system.d/
+cp <builddir>/share/dbus-1/system.d/org.containers.bluechi.Agent.conf /etc/dbus-1/system.d/
+cp <builddir>/share/dbus-1/system.d/org.containers.bluechi.conf /etc/dbus-1/system.d/
```

Comment:
`org.containers.hirte.` should be replaced with `io.github.eclipse-bluechi.bluechi` according to the doc and today's discussion